commons/types/transaction.go

```diff
@@ -210,14 +215,15 @@ func ToTransactions(txn *badger.Txn) ([]*Transaction, error) {
 	return transactions, nil
 }
 
-func TransactionPaging(txn *badger.Txn, startingHash string, page, pageSize int) ([]*Transaction, error){
+func TransactionPaging(txn *badger.Txn, startingHash string, page, pageSize int) (response, error) {
```

Comment:
Changing the return value here; also need to change dapos_api.go?

NOTE: We should not change the structure of the `data` object returned (it needs to remain an array of Tx). Instead, the `startingHash` value should be returned as a peer of `data`. Ex; as part of the `Response` structure when the value is not null.